commit a915655764089cf7e292d76ce3fc9cdbbc4f88f0
Author: q66 <q66@chimera-linux.org>
Date:   Mon Oct 3 01:56:45 2022 +0200

    drop -latomic for riscv, we don't need or want it

diff --git a/Makefile.am b/Makefile.am
index d64df5d..ba6cd48 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -41,9 +41,6 @@ memhog_LDADD = libnuma.la
 
 libnuma_la_SOURCES = libnuma.c syscall.c distance.c affinity.c affinity.h sysfs.c sysfs.h rtnetlink.c rtnetlink.h versions.ldscript
 libnuma_la_LDFLAGS = -version-info 1:0:0 -Wl,--version-script,$(srcdir)/versions.ldscript -Wl,-init,numa_init -Wl,-fini,numa_fini
-if RISCV64
-libnuma_la_LDFLAGS += -latomic
-endif
 
 check_PROGRAMS = \
 	test/distance \
